src/Dapr.Actors/DaprHttpInteractor.cs

```diff
@@ -113,6 +113,12 @@ HttpRequestMessage RequestFunc()
 
                 request.Headers.Add(Constants.RequestHeaderName, Encoding.UTF8.GetString(serializedHeader, 0, serializedHeader.Length));
 
+                var reentrancyId = Helper.GetReentrancyContext();
+                if (reentrancyId != string.Empty)
```

Comment:
Use `null` as a sentinel value, not empty string. It's a better practice in general to use `null` as a sentinel for *unset* because then if you have a bug you will get a stack trace instead of wierd behavior.